src/YAXArrays.jl

```diff
@@ -1,4 +1,15 @@
 module YAXArrays
+
+"""
+Default configuration for YAXArrays, has the following fields:
+
+- `workdir[]::String = "./"` The default location for temporary cubes.
+- `recal[]::Bool = false` set to true if you want `@loadOrGenerate` to always recalculate the results.
+- `chunksize[]::Any = :input` TODO
+- `max_cache[]::Float64 = 1e8` The maximum cache used by mapCube.
+- `cubedir[]::""` the default location for `Cube()` without an argument.
+- `subsetextension::Array{Any} = []` TODO
```

Comment:
```suggestion
- `subsetextensions::Array{Any} = []` List of registered functions, that convert subsetting input into dimension boundaries. 
```

Reply:
This is used in EarthDataLab to register the replaceregion function to enable spatial subsampling with region names. 
https://github.com/JuliaDataCubes/EarthDataLab.jl/blob/ee0e13f7b5560d0d8638c2e20fcfd24281393b46/src/EarthDataLab.jl#L39
I am not sure, whether we should add this example.